swagger/public_v1/api_spec.json

```diff
@@ -394,6 +394,30 @@
             "type": "string",
             "description": "Generated summary of the course.",
             "example": "PGCE with QTS full time"
+          },
+          "provider_type": {
+            "type": "string",
+            "description": "The type of provider.",
+            "example": "scitt",
+            "enum":[
+              "scitt",
+              "lead_school",
+              "university",
+              "unknown",
+              "invalid_value"
+            ]
+          },
+          "provider_code": {
+            "type": "string",
+            "description": "Code that uniquely identifies this provider.",
+            "maxLength": 3,
+            "minLength": 3,
+            "example": "X99"
+          },
+          "description": {
+            "type": "string",
+            "description": "A combined string of qualification, study mode & programme type.",
+            "example": "PGCE with QTS full time teaching apprenticeship."
```

Comment:
@misaka should this be here? as it is a calculated value and should be off loaded to the client side

Reply:
Ya, I remember we had a discussion about this, and I think we wanted to have a standard way of summarising the course. We even suggested that maybe we want to call this `summary`, which might make more sense than `description`.

Reply:
@defong Do we currently produce this in API v3?

Reply:
It's rooted here https://github.com/DFE-Digital/teacher-training-api/blob/1cff52b7d5283c2fe2d9ba4f9094ebd0524c4a54/app/models/course.rb#L362-L366

Reply:
@misaka @defong - have you come to a decision on the above?

Reply:
> Ya, I remember we had a discussion about this, and I think we wanted to have a standard way of summarising the course. We even suggested that maybe we want to call this `summary`, which might make more sense than `description`.

https://github.com/DFE-Digital/teacher-training-api/blob/f79edc40b78c8d844906f4e3239d315310649fd5/swagger/public_v1/api_spec.json#L393-L397

https://github.com/DFE-Digital/teacher-training-api/blob/2d9c96133190f82bce664de1a459b3f2a9c6976e/app/serializers/api/public/v1/serializable_course.rb#L110-L112

:facepalm: